bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/ResourceLimiter.java

```diff
@@ -89,6 +91,22 @@ public int getMaxInFlightRpcs() {
   }
 
   /**
+   * <p>Getter for the field <code>currentInFlightRpcs</code>.</p>
+   *
+   * @return The current number of allowed in-flight RPCs
+   */
+  public int getCurrentInFlightRpcs() {
```

Comment:
The name of the method doesn't seem to go with the comment. Is it allowed RPCs or currently executing RPCs?